app/validators/expiry_date_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ExpiryDateValidator < ActiveModel::Validator
+  def validate(record)
+    if !record.expires_at.nil? && record.expires_at <= DateTime.now
```

Comment:
On this line you are checking if record.expires_at is present or not nil.
Since nil is falsy in ruby we can just check `if record.expires_at && record.expires_at <= DateTime.now`